src/module/page_with_sidebar/page_with_sidebar.directive.coffee

```diff
@@ -12,22 +12,25 @@ class GlPageWithSidebar extends Directive
 
 class _glPageWithSidebar extends Controller
     constructor: (@$scope, glMenuService, @$timeout, @$window) ->
-        @sidebarPinned = @$window.innerWidth > 800
```

Comment:
please don't remove the feature that auto unpin if the windows is < 800 (tablet mode)

Reply:
OK.  I think it would make sense if it used `@sidebarPinned = @$window.innerWidth > 800` as a default only if there's currently no saved value in `@$window.localStorage.sidebarPinned`.  Do you agree that the saved value should take precedence?

Reply:
Also, do you know if it's possible to invoke a browser refresh from within the `.spec.coffee` script?  That would be useful for testing some of this.  I'm not sure whether a browser automation API is accessible from those test scripts.

Reply:
I think it makes sense to have a sensible default value according to screen size.

The saved value should of-course take precedence, but here the default value, will be undefined, which is evaluate to false thus unpinned.

You can't do a browser refresh in unit tests. but you can set the cookie from within the test, and verify that the cookie has been set